README.md

```diff
@@ -1,3 +1,4 @@
+[![GoDoc](https://godoc.org/github.com/tomponline/beanstalkworker/beanstalkworker?status.svg)](https://godoc.org/github.com/tomponline/beanstalkworker/beanstalkworker)
```

Comment:
this might need to be updated when https://github.com/tomponline/beanstalkworker/pull/31 will get merged, as there is a change in the package path

Reply:
@NevilleC agreed, will do